src/collector.ts

```diff
@@ -71,6 +71,73 @@ class DependencyCollector implements IDependencyCollector {
     }
 }
 
+class NaivePyParser {
+    constructor(objSam: any) {
+        this.objSam = objSam;
+        this.parser = this.createPyParser()
+    }
+
+    objSam: any;
+    parser: Xml2Object;
+    dependencies: Array<IDependency> = [];
+    isDependency: boolean = false;
+    versionStartLine: number = 0;
+    versionStartColumn: number = 0;
```

Comment:
I think these two fields `versionStartLine` and `versionStartColumn` are not used anywhere?